lib/commons/dom/get-rect-stack.js

```diff
@@ -360,6 +360,8 @@ function addNodeToGrid(grid, vNode) {
     const endRow = ((y + rect.height) / gridSize) | 0;
     const endCol = ((x + rect.width) / gridSize) | 0;
 
+    grid._numCols = Math.max(grid._numCols ?? 0, endCol);
```

Comment:
This feels a little off. If `_numCols` needs to be different, it should be fixed where that value is set. If it doesn't need to be changed, this should probably just be a local variable.